lineage/states/StateDistributionGamma.py

```diff
@@ -68,7 +68,7 @@ def pdf(self, x: np.ndarray):
         # Update for observed Bernoulli
         ll[np.isfinite(x[:, 0])] += sp.bernoulli.logpmf(x[np.isfinite(x[:, 0]), 0], self.params[0])
 
-        # Make sure censored values are censored
+        # likelihood of negative values should be zero
         ll[x[:, 1] < 0] = 0.0
 
         return np.exp(ll)
```

Comment:
It might be useful to make a logpdf function in this class, since you probably want that most of the time. The pdf method can just call logpdf then take the exp.